Examples/MAX32650/ADC_MAX11261/.vscode/settings.json

```diff
@@ -67,7 +67,10 @@
         "${config:MAXIM_PATH}/Libraries/MiscDrivers"
     ],
     "C_Cpp.default.defines": [
-        "${config:board}"
+        "${config:board}",
+        "TARGET=${config:target}",
+        "TARGET_REV=0x4131",
+        "__GNUC__"
```

Comment:
I don't think we have any code anymore that uses 'TARGET_REV'.  We should probably start trying to phase it out.